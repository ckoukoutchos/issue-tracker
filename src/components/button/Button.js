import React from 'react';
import styles from './Button.module.css';

const Button = (props) => {
  const { children, type } = props;

  return <button className={styles.button} type={type}>{children}</button>
}

export default Button;