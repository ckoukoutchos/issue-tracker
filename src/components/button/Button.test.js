import React from 'react';
import Button from './Button';
import renderer from 'react-test-renderer';

it('renders the Button component correctly', () => {
  const tree = renderer
    .create(<Button type="submit">Submit</Button>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});