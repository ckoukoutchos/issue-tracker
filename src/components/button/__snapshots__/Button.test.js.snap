// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Button component correctly 1`] = `
<button
  className="button"
  type="submit"
>
  Submit
</button>
`;
